core/src/main/java/org/pegdown/ParserWithDirectives.java

```diff
@@ -162,7 +162,7 @@ public Node parseBlockContents(Var<Integer> markerLength, StringBuilderVar block
         // recursively parse inner contents
         Context<Object> context = getContext();
         Integer length = markerLength.get();
-        String contents = block.getString();
+        String contents = "\n" + block.getString() + "\n";
```

Comment:
If the concern is mostly around callout-style markups, would it make sense to implement a utility function that the custom markups can use to opt into blank line leniency?